<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#dashwidget{
	background-color : qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(254, 246, 239, 255), stop:0.766169 rgba(255, 222, 222, 255))
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="dashwidget" native="true">
     <property name="minimumSize">
      <size>
       <width>1400</width>
       <height>750</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#dashwidget{
	background-color : qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(254, 246, 239, 255), stop:0.766169 rgba(255, 222, 222, 255))
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>750</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(75, 62, 87);</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="data1">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color:rgb(216, 196, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="data2">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color:rgb(216, 196, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="data3">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color:rgb(216, 196, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="data4">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color:rgb(216, 196, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="dash">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Calibri&quot;;
color: rgb(190, 159, 221);</string>
             </property>
             <property name="text">
              <string>BACK TO DASH</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="logout">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;Calibri&quot;;
color:rgb(190, 159, 221);
background-color:rgb(75, 62, 87);</string>
             </property>
             <property name="text">
              <string> LOGOUT</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Lib-Management-latest/Lib-Management-Latest-main/logout icon.png</normaloff>../Lib-Management-latest/Lib-Management-Latest-main/logout icon.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>100</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="libraryRecords">
            <property name="minimumSize">
             <size>
              <width>1035</width>
              <height>54</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 30px;
font: 75 26pt &quot;Trebuchet MS&quot;;
color: rgb(255, 85, 127);</string>
            </property>
            <property name="text">
             <string>ADD A NEW PATIENT</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="value">
            <property name="minimumSize">
             <size>
              <width>1035</width>
              <height>27</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;Calibri&quot; bold;
color:rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="spacing">
             <number>7</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color:rgb(75, 62, 87);
font: 14pt &quot;Calibri&quot;;
margin : 5px;
</string>
              </property>
              <property name="text">
               <string>gestational age in weeks</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="heatout1">
              <property name="minimumSize">
               <size>
                <width>684</width>
                <height>41</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0, 0, 0, 0);
border:1px solid rgb(0, 55, 80);
color: rgb(0, 55, 80);
font: 75 12pt &quot;Calibri&quot; bold;

</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="weight1">
              <property name="minimumSize">
               <size>
                <width>684</width>
                <height>41</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0, 0, 0, 0);
border:1px solid rgb(0, 55, 80);
color: rgb(0, 55, 80);
font: 75 12pt &quot;Calibri&quot; bold;

</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color:rgb(75, 62, 87);
font: 14pt &quot;Calibri&quot;;
margin : 5px;</string>
              </property>
              <property name="text">
               <string>heatout of incubator</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="age1">
              <property name="minimumSize">
               <size>
                <width>684</width>
                <height>41</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0, 0, 0, 0);
border:1px solid rgb(0, 55, 80);
color: rgb(0, 55, 80);
font: 75 12pt &quot;Calibri&quot; bold;

</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QFormLayout" name="formLayout"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">color:rgb(75, 62, 87);
font: 14pt &quot;Calibri&quot;;
margin : 5px;</string>
              </property>
              <property name="text">
               <string>room temp</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color:rgb(75, 62, 87);
font: 14pt &quot;Calibri&quot;;
margin : 5px;</string>
              </property>
              <property name="text">
               <string>weight in grams</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="temp">
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0, 0, 0, 0);
border:1px solid rgb(0, 55, 80);
color: rgb(0, 55, 80);
font: 75 12pt &quot;Calibri&quot; bold;
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">color:rgb(75, 62, 87);
font: 14pt &quot;Calibri&quot;;
margin : 5px;</string>
              </property>
              <property name="text">
               <string>room humidity</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="hum">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0, 0, 0, 0);
border:1px solid rgb(0, 55, 80);
color: rgb(0, 55, 80);
font: 75 12pt &quot;Calibri&quot; bold;
</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>150</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="predict">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 15pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
background-color:rgb(255, 179, 179);
border-radius:20px;


</string>
            </property>
            <property name="text">
             <string>Predict</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
